Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfig.swift

```diff
@@ -16,6 +16,18 @@ enum ConfigPaymentMethodType: String, Codable {
     case googlePay = "GOOGLE_PAY"
     case goCardlessMandate = "GOCARDLESS"
     case klarna = "KLARNA"
+    case payNlIdeal = "PAY_NL_IDEAL"
+    
+    case unknown
+    
+    var isEnabled: Bool {
```

Comment:
Let's leave that as a quick fix, but create a ticket to abstract it. Shouldn't this be set with config API call?

Reply:
Not sure I follow.. Could you clarify what you want to do?

Reply:
What I mean is, shouldn't all other payment methods configured on the dashboard be available? Except for GooglePay of course. Talking about the `isEnabled` bool. @ceriksson